### PR TITLE
Implement CI testing for BIDS

### LINKED ISSUE #1: Implement CI testing for BIDS

### PR DESCRIPTION
**Pull request by alexfoias**
Fixes https://github.com/spine-generic/data-single-subject/issues/1

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+# blocklist
+branches:
+  except:
+  - master
+
+# safelist
+branches:
+  only:
+  - af/acq_param_ci
```
Comment (jcohenadad) [inline .travis.yml:17]: probably needs to become master, no?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  - sudo rm -rf ~/.nvm
+  - curl -sL https://deb.nodesource.com/setup_current.x | sudo -E bash -
+  - sudo apt-get install -y nodejs
+  - sudo npm install -g bids-validator
+
+# install spine-generic for acquisition parameters check 
+install:
+  - git clone https://github.com/spine-generic/spine-generic.git code/spine-generic
+  - cd code/spine-generic
+  - git checkout af/acq_param_ci
```
Comment (jcohenadad) [inline .travis.yml:30]: to be updated

File: README.md [hunk 3]
```diff
@@ -1,4 +1,5 @@
 # Spine Generic Public Database (Single Subject)
+[![Build Status](https://travis-ci.com/spine-generic/data-single-subject.svg?branch=af%2Facq_param_ci)](https://travis-ci.com/spine-generic/data-single-subject)
```
Comment (jcohenadad) [inline README.md:2]: update branch

**Comment by alexfoias** [comment]
my mistake. we can close this PR because it's a duplicate with https://github.com/spine-generic/data-single-subject/pull/3

**Pull request closed without merging**